src/llmq/signing_shares.cpp

```diff
@@ -1728,8 +1728,9 @@ void CSigSharesManager::ForceReAnnouncement(const CQuorumCPtr& quorum, Consensus
 
 MessageProcessingResult CSigSharesManager::HandleNewRecoveredSig(const llmq::CRecoveredSig& recoveredSig)
 {
+    auto signHash = recoveredSig.buildSignHash().Get();
```

Comment:
nit: there's no reason to put `auto` everywhere.

```suggestion
    uint256 signHash{recoveredSig.buildSignHash().Get()};
```